flag_int64_slice.go

```diff
@@ -22,7 +22,12 @@ func (f *Int64Slice) Set(value string) error {
 
```

Comment:
Commenting at the top because there's nowhere else to put it—

It's not called out in the original issue, but there is also a `float64` slice that probably has the same issue here.

Reply:
Float64 Slice is not on the v1